feature/experimental/isis/ate_tests/isis_interface_passive_test/isis_interface_passive_test.go

```diff
@@ -0,0 +1,399 @@
+				t.Errorf("FAIL- Expected v6 unicast metric value not found,got %d,want %d", got, v6_metric)
+			}
+		})
+		t.Run("Adjacency state checks", func(t *testing.T) {
+			adjPath := statePath.Interface(intfName).Level(2).Adjacency(ateSysID)
+
+			// Checking neighbor sysid.
+			if got := gnmi.Get(t, dut, adjPath.SystemId().State()); got != ateSysID {
+				t.Errorf("FAIL- Expected neighbor system id not found,got %s,want %s", got, ateSysID)
+			}
+			// Checking isis area address.
+			want := []string{ateAreaAddress, dutAreaAddress}
+			if got := gnmi.Get(t, dut, adjPath.AreaAddress().State()); !cmp.Equal(got, want, cmpopts.SortSlices(func(a, b string) bool { return a < b })) {
+				t.Errorf("FAIL- Expected area address not found,got %s,want %s", got, want)
+			}
+			// Checking dis system id.
+			if got := gnmi.Get(t, dut, adjPath.DisSystemId().State()); got != "0000.0000.0000" {
+				t.Errorf("FAIL- Expected dis system id not found,got %s,want %s", got, "0000.0000.0000")
+			}
+			// Checking isis local extended circuit id.
+			if got := gnmi.Get(t, dut, adjPath.LocalExtendedCircuitId().State()); got == 0 {
+				t.Errorf("FAIL- Expected local extended circuit id not found,expected non-zero value,got %d", got)
+			}
+			// Checking multitopology.
+			if got := gnmi.Get(t, dut, adjPath.MultiTopology().State()); got != false {
+				t.Errorf("FAIL- Expected value for multi topology not found,got %t,want %t", got, false)
+			}
+			// Checking neighbor circuit type.
+			if got := gnmi.Get(t, dut, adjPath.NeighborCircuitType().State()); got != oc.Isis_LevelType_LEVEL_2 {
+				t.Errorf("FAIL- Expected value for circuit type not found,got %s,want %s", got, oc.Isis_LevelType_LEVEL_2)
+			}
+			// Checking neighbor ipv4 address.
+			if got := gnmi.Get(t, dut, adjPath.NeighborIpv4Address().State()); got != atePort1attr.IPv4 {
+				t.Errorf("FAIL- Expected value for ipv4 address not found,got %s,want %s", got, atePort1attr.IPv4)
+			}
+			// Checking isis neighbor extended circuit id.
+			if got := gnmi.Get(t, dut, adjPath.NeighborExtendedCircuitId().State()); got == 0 {
+				t.Errorf("FAIL- Expected neighbor extended circuit id not found,expected non-zero value,got %d", got)
+			}
+			// Checking neighbor snpa.
+			snpa_address := gnmi.Get(t, dut, adjPath.NeighborSnpa().State())
+			mac, err := net.ParseMAC(snpa_address)
+			if !(mac != nil && err == nil) {
+				t.Errorf("FAIL- Expected value for snpa address not found,got %s", snpa_address)
+			}
+			// Checking isis adjacency address families.
+			if got := gnmi.Get(t, dut, adjPath.Nlpid().State()); !cmp.Equal(got, []oc.E_Adjacency_Nlpid{oc.Adjacency_Nlpid_IPV4, oc.Adjacency_Nlpid_IPV6}) {
+				t.Errorf("FAIL- Expected address families not found,got %s,want %s", got, []oc.E_Adjacency_Nlpid{oc.Adjacency_Nlpid_IPV4, oc.Adjacency_Nlpid_IPV6})
+			}
+			// Checking neighbor ipv6 address.
+			ipv6_address := gnmi.Get(t, dut, adjPath.NeighborIpv6Address().State())
```

Comment:
remove all the underscores from these names and use camelCase instead

Reply:
taken care in latest commit